aviary/examples/multi_mission/run_multimission_example_large_single_aisle.py

```diff
@@ -0,0 +1,328 @@
+        for prob in self.probs:
+            prob.model.options['aviary_options'] = prob.aviary_inputs
+            prob.model.options['aviary_metadata'] = prob.meta_data
+            prob.model.options['phase_info'] = prob.phase_info
+
+        # Aviary's problem setup wrapper uses these ignored warnings to suppress
+        # some warnings related to variable promotion. Replicating that here with
+        # setup for the super problem
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore", om.OpenMDAOWarning)
+            warnings.simplefilter("ignore", om.PromotionWarning)
+            self.setup(check='all')
+
+    def run(self):
+        self.model.set_solver_print(0)
+        dm.run_problem(self, make_plots=False)
+
+    def get_design_range(self):
+        """Finds the longest mission and sets its range as the design range for all
+            Aviary problems. Used within Aviary for sizing subsystems (avionics and AC)."""
+        design_range = []
+        for phase_info in self.phase_infos:
+            design_range.append(phase_info['post_mission']
+                                ['target_range'][0])  # TBD add units
+        design_range_min = np.min(design_range)
+        design_range_max = np.max(design_range)
+        return design_range_max, design_range_min  # design_range_min
+
+    def create_timeseries_plots(self, plotvars=[], show=True):
+        """
+        Temporary create plots manually because graphing won't work for dual-trajectories.
+        Creates timeseries plots for any variables within timeseries. Specify variables
+        and units by setting plotvars = [('altitude','ft')]. Any number of vars can be added.
+        """
+        plt.figure()
+        for plotidx, (var, unit) in enumerate(plotvars):
+            plt.subplot(int(np.ceil(len(plotvars)/2)), 2, plotidx+1)
+            for i in range(self.num_missions):
+                time = np.array([])
+                yvar = np.array([])
+                # this loop concatenates data from all phases
+                for phase in self.phases[f"{self.group_prefix}_{i}"]:
+                    rawt = self.get_val(
+                        f"{self.group_prefix}_{i}.traj.{phase}.timeseries.time",
+                        units='s')
+                    rawy = self.get_val(
+                        f"{self.group_prefix}_{i}.traj.{phase}.timeseries.{var}",
+                        units=unit)
+                    time = np.hstack([time, np.ndarray.flatten(rawt)])
+                    yvar = np.hstack([yvar, np.ndarray.flatten(rawy)])
+                plt.plot(time, yvar, linewidth=self.num_missions-i)
+            plt.xlabel("Time (s)")
+            plt.ylabel(f"{var.title()} ({unit})")
+            plt.grid()
+        plt.figlegend([f"Mission {i}" for i in range(self.num_missions)])
+        if show:
+            plt.show()
+
+    def create_payload_range_plot(self, show=True):
```

Comment:
This isn't a payload-range diagram, that can only be built using off-design missions (with range as a fall-out). I say either remove this plot, or change it to points on the diagram and not lines, and remove the 0 range point.

The design missions are just points _inside_ the viable area of the payload-range diagram and don't necessarily define its boundaries.

Reply:
Yeah It's been removed. I was thinking the same thing.